daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1140,6 +1194,17 @@ message Update {
     Expr key = 2;
   }
 
+  // Try-catch block.
+  message TryCatch {
+    Type return_type = 1;
+    Expr try = 2;
```

Comment:
I'm not sure if `try` is a keyword in Scala and I don't know what would happen if it was. Let's err on the safe side and make sure we get something with a predictable name:
```suggestion
    Expr try_expr = 2;
```
Similarly for `catch`.

Reply:
Scala keyword is not an issue, for instance we have some fileds named `type`, and this is not an issue for the Scala decoder.
However I do not like `try`, I would prefer `expr` or `body`

Reply:
I changed them to `try_expr` and `catch_expr`